RealEstate/apps/core/static/style.css

```diff
@@ -251,3 +245,105 @@ a.mail {
   width:inherit;
   height:100%;
 }
+
+/* report */
+.finalGrade-ref{
+    font-weight: bold;
+    font-size: 64px;
+    vertical-align: middle;
+}
+#finalGrade-Ap{
+    color: #00CC00;
+}
+#finalGrade-A{
+    color: #009900;
+}
+#finalGrade-B{
+    color: #CCCC00;
+}
+#finalGrade-C{
+    color: #FF9900;
+}
+#finalGrade-D{
+    color: #FF6600;
+}
+#finalGrade-F{
+    color: #FF0000;
+}
+
+#padding-after{
+	padding-bottom: 50px;
+}
+
+#report-title{
+	text-decoration: underline;
+}
+
+#legend{
+    outline: solid;
+    outline-color: #bbb;
+    outline-width: 1px;
+    min-height:100px;
+}
+.row-height{
+	max-height:250px;
+}
+#hb-1{
+	background-color:rgba(217, 95, 73, 0.9);
+}
+#hb-2{
+	background-color:rgba(237, 194, 51, 0.9);
+}
+.weight-chart{
+	min-height:400px;
+	width: 100%;
+}
+
+#fork {
+    position: absolute;
+    top: 0;
+    right: 0;
+    border: 0;
+}
+
+.legend {
+    width: 10em;
+    border: 1px solid black;
```

Comment:
I added a new couple with just the default categories and one house.  This border did not expand to fit around all of the content (for the Weighting tab).  It looked fine without it.  I think we can remove it